var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -80,6 +82,9 @@ def configure_args(self):
         if sys.platform == 'darwin':
             args.append('--without-ns')
 
+        if '+native' in spec:
+            args.append('--with-native-compilation')
```

Comment:
When I try to set +native for emacs@27.1, I get:
```
configure: WARNING: unrecognized options: --with-native-compilation
```
Is this limited to some newer versions?

Reply:
Yes it's only restricted to master branch. And should come in next release.

Reply:
Ah, now I also see it in the title, sorry. :smile: Maybe it should be restricted to @master, then?